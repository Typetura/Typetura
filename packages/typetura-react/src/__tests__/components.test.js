import React from 'react';
import renderer from 'react-test-renderer';

import Typetura from '../';

jest.mock('typeturajs');

describe('PrimaryHeadline', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Typetura.PrimaryHeadline>PrimaryHeadline</Typetura.PrimaryHeadline>);

    expect(tree).toMatchSnapshot();
  });
});
describe('PrimarySubheadline', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Typetura.PrimarySubheadline>PrimarySubheadline</Typetura.PrimarySubheadline>);

    expect(tree).toMatchSnapshot();
  });
});
describe('SectionHeadline', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Typetura.SectionHeadline>SectionHeadline</Typetura.SectionHeadline>);

    expect(tree).toMatchSnapshot();
  });
});
describe('SectionSubheadline', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Typetura.SectionSubheadline>SectionSubheadline</Typetura.SectionSubheadline>);

    expect(tree).toMatchSnapshot();
  });
});
describe('Meta', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Typetura.Meta>Meta</Typetura.Meta>);

    expect(tree).toMatchSnapshot();
  });
});
describe('SectionLabel', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Typetura.SectionLabel>SectionLabel</Typetura.SectionLabel>);

    expect(tree).toMatchSnapshot();
  });
});
describe('Big', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Typetura.Big>Big</Typetura.Big>);

    expect(tree).toMatchSnapshot();
  });
});
describe('Small', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Typetura.Small>Small</Typetura.Small>);

    expect(tree).toMatchSnapshot();
  });
});
describe('Caption', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Typetura.Caption>Caption</Typetura.Caption>);

    expect(tree).toMatchSnapshot();
  });
});
describe('Blockquote', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Typetura.Blockquote>Blockquote</Typetura.Blockquote>);

    expect(tree).toMatchSnapshot();
  });
});
describe('Pullquote', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Typetura.Pullquote>Pullquote</Typetura.Pullquote>);

    expect(tree).toMatchSnapshot();
  });
});
describe('Context', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Typetura.Context>Context</Typetura.Context>);

    expect(tree).toMatchSnapshot();
  });
});
