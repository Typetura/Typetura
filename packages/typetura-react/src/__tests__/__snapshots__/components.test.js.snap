// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Big renders correctly 1`] = `
<p
  className="big"
>
  Big
</p>
`;

exports[`Blockquote renders correctly 1`] = `
<blockquote
  className="blockquote"
>
  Blockquote
</blockquote>
`;

exports[`Caption renders correctly 1`] = `
<p
  className="caption"
>
  Caption
</p>
`;

exports[`Context renders correctly 1`] = `
<div>
  Context
</div>
`;

exports[`Meta renders correctly 1`] = `
<h3
  className="meta"
>
  Meta
</h3>
`;

exports[`PrimaryHeadline renders correctly 1`] = `
<h1
  className="primary-headline"
>
  PrimaryHeadline
</h1>
`;

exports[`PrimarySubheadline renders correctly 1`] = `
<h2
  className="primary-subheadline"
>
  PrimarySubheadline
</h2>
`;

exports[`Pullquote renders correctly 1`] = `
<p
  className="pullquote"
>
  Pullquote
</p>
`;

exports[`SectionHeadline renders correctly 1`] = `
<h1
  className="section-headline"
>
  SectionHeadline
</h1>
`;

exports[`SectionLabel renders correctly 1`] = `
<h2
  className="section-label"
>
  SectionLabel
</h2>
`;

exports[`SectionSubheadline renders correctly 1`] = `
<h2
  className="section-subheadline"
>
  SectionSubheadline
</h2>
`;

exports[`Small renders correctly 1`] = `
<p
  className="small"
>
  Small
</p>
`;
